modules/reference/pages/admin/admin-menu.adoc

```diff
@@ -1,8 +1,10 @@
 = Admin Menu
 
-The menu:Main Menu[Admin] pages allows {productname} customers to manage the basic configuration, including creating and managing multiple organizations.
+The [guimenu]``Admin`` menu provides access to features for managing {productname} configuration.
```

Comment:
```suggestion
The menu:Admin[] menu provides access to features for managing {productname} configuration.
```

Reply:
I thought we decided against doing this?

Reply:
Maybe, you are right. Only if there are more than one entries using menu: ? I think we did not talk abut this (maybe, I forgot it).

Reply:
I might have imagined it :wink: